test/include/test.h

```diff
@@ -0,0 +1,26 @@
+#ifndef TEST_TEST_H
+#define TEST_TEST_H
+
+#include <kernel/com.h>
+#include <stdbool.h>
+
+#define FAIL {serial_writestring("Test terminated\n"); *(error_count)++; return;}
+#define PASS {serial_writestring("Test passed\n"); *(error_count) = 0; return;}
+
+#define ASSERT(x, y) if (x == y) {} \
+	else { serial_writestring("Assert fail on "); serial_writestring(__FILE__); /* serial_writestring(__LINE__); to be replaced with sprintf*/ (*error_count)++; return;}
```

Comment:
Jedynie takie zmianki kosmetyczne małe:
```suggestion
	else {serial_writestring("Assert fail on "); serial_writestring(__FILE__); /* serial_writestring(__LINE__); to be replaced with sprintf*/ (*error_count)++; return;}
```